src/main/common/maths.c

```diff
@@ -118,16 +118,22 @@ float powerf(float base, int exp) {
     return result;
 }
 
-int32_t applyDeadband(int32_t value, int32_t deadband)
+int32_t applyDeadband(const int32_t value, const int32_t deadband)
 {
     if (ABS(value) < deadband) {
```

Comment:
The one problem I see here is that this will have different results to the original function if `deadband < 0`. Just replacing it without checking that this doesn't matter everywhere where it's used is problematic.

Reply:
@mikeller : I didn't think about negative deadband ... it may be useful sometimes ..

Reply:
I don't unstandard the concept of negative deadband, what is the resulting value changes sign?

Reply:
BTW: It would be beter to alter test below to handle 0 as positive ...

Reply:
Ok, so to sum this up: